index.html

```diff
@@ -15,16 +15,20 @@
   <body>
     <header class="container">
       <h1><a href="#">Bekjon</a></h1>
-      <nav class="hamburger">
+      <div class="hamburger">
         <div class="line"></div>
         <div class="line"></div>
         <div class="line"></div>
+      </div>
+      <nav>
         <ul class="nav-links">
-          <li class="nav-link">Portfolio</li>
-          <li class="nav-link">About</li>
-          <li class="nav-link">Contact</li>
+          <li class="nav-link"><a href="#works">Portfolio</a></li>
+          <li class="nav-link"><a href="#about">About</a></li>
+          <li class="nav-link"><a href="#contact">Contact</a></li>
           <li class="nav-link">
-            <i class="far fa-envelope"></i>
```

Comment:
- [ ] According to the project requirement you should stick to the design as much as possible. Please fix the navbar of the desktop version. See the image below for reference.
### Your design
![his](https://user-images.githubusercontent.com/81727988/217499773-7131e48d-d223-404b-9101-3e38d8beb582.png)

### Figma design
![deskfimga](https://user-images.githubusercontent.com/81727988/217499799-02ad745f-c57f-4fc6-9aca-0b7e809a6e0d.png)